rust-toolchain

```diff
@@ -1 +0,0 @@
-nightly-2019-07-31
```

Comment:
why delete this?

Reply:
The submit was accidental, will put it back in again. I have to delete this locally at my pc as my rust does not want to compile with this in there.

Reply:
Update your toolchain.

The Bulletproofs crate requires this version